SPECS-EXTENDED/buildah/buildah.spec

```diff
@@ -32,7 +32,7 @@ BuildRequires:  btrfs-progs-devel
 BuildRequires:  device-mapper-devel
 BuildRequires:  git
 BuildRequires:  glib2-devel
-BuildRequires:  glibc-static >= 2.35-6%{?dist}
+BuildRequires:  glibc-static >= 2.38-1%{?dist}
```

Comment:
Do we even need the qualifier?

Reply:
Yes, we need to ensure that these packages are rebuilt/re-released with the latest glibc static components, and the qualifier is needed for that. More info in #3748